onnxruntime/core/providers/webgpu/reduction/reduction_ops.h

```diff
@@ -13,22 +13,23 @@ namespace webgpu {
 // reduceOpSpecificCode is a 3-element array of strings that represent the op specific code for the reduce operation.
 // The first element is the loop header, the second element is the loop body, and the third element is the loop footer.
 // The loop header is the code that is executed before the loop starts. The loop body is the code that is executed for each element in the loop.
-// The loop footer is the code that is executed after the loop ends.
+// The loop footer is the code that is executed after the loop ends. The loop body should contain the code that accumulates the result of the reduction and
+// the loop footer should contain the code that assigins output_value the result of the reduction.
 typedef std::array<std::string, 3> ReduceOpSpecificCode;
```

Comment:
suggest to change this to struct for better readability
```
struct ReduceOpSpecificCode {
  std::string header;
  std::string loop_body;
  std::string footer;
}
```

Reply:
Yeah. I considered this but I thought using an array of strings may be more efficient.

Reply:
If the "efficient" here refers to runtime latency (CPU cycles), it does not really make difference between struct and array for 3 std::string variables for C++.